panels/site_sections/panel_app_management.py

```diff
@@ -15,52 +15,40 @@ def app(self, session, id, message='', csrf_token='', explanation=None):
             'app': session.panel_application(id)
         }
 
-    def mark(self, session, status, **params):
+    def form(self, session, message='', **params):
         app = session.panel_application(params)
-        if app.status != c.PENDING:
-            raise HTTPRedirect('index?message={}{}', 'That panel was already marked as ', app.status_label)
-
-        app.status = int(status)
-        create_group = len(app.applicants) - len(app.matching_attendees) > 1 and not getattr(app.submitter.matching_attendee, 'group_id', None)
```

Comment:
I ripped out the bulk auto-create that was implemented here.  After talking it over with the panel folks, it's not how they want to do things this year.  Instead of happening automatically when a panel is accepted, there will be a separate admin form for this kind of thing which will be added later.